xen/xenops_types.ml

```diff
@@ -135,6 +135,7 @@ module Vm = struct
     framebuffer_ip: string option [@default None];
     vncterm: bool [@default true];
     vncterm_ip: string option [@default None];
+    pci_passthrough: bool [@default false]
```

Comment:
I assume we don't have to worry about migration here because migration with PCI devices was never supported before.